src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -8239,6 +8239,7 @@ public partial class UnobservedTaskExceptionEventArgs : System.EventArgs
         public bool Observed { get { throw null; } }
         public void SetObserved() { }
     }
+    [System.Runtime.CompilerServices.AsyncMethodBuilderAttribute(typeof(System.Runtime.CompilerServices.AsyncValueTaskMethodBuilder))]
```

Comment:
This attribute is required to avoid compiler errors (for verifying that async methods return a task-like type), but it is being excluded here:
https://github.com/dotnet/corefx/blob/26b932e3c56702382d13ece010b300d3181d2891/eng/DefaultGenApiDocIds.txt#L53

```text
error CS1983: The return type of an async method must be void, Task, Task<T>, a task-like type, IAsyncEnumerable<T>, or IAsyncEnumerator<T>
```

Reply:
https://github.com/dotnet/corefx/issues/35602